app/src/main/java/org/covidwatch/android/ble/BluetoothManager.kt

```diff
@@ -58,7 +58,7 @@ class BluetoothManagerImpl(
                         foregroundNotification()
                     )
                 )
-                if (BluetoothAdapter.getDefaultAdapter().bluetoothLeAdvertiser != null) start()
+                if (BluetoothAdapter.getDefaultAdapter()?.bluetoothLeAdvertiser != null) start()
```

Comment:
`getDefaultAdapter()` might return null because of devices w/o bluetooth
this also fixes a crash on emulators